nikola/plugins/command/new_post.py

```diff
@@ -213,6 +213,14 @@ class CommandNewPost(Command):
                     'bbcode, html, textile, txt2tags',
         },
         {
+            'name': 'kernel',
```

Comment:
yes... we need to specify the kernel to build the correct notebook template with the correct kernel specification... if not... the notebook is no validated ok and will fail...
I am open to ideas about how to deal with this that, as you said, is not available for other compilers...
One option would be probably make it only available with ipynb (warn and exit if you try to use it with other compilers... but maybe they are other options...